### DISCUSSION TITLE
购买了两次的专业版，但是不能用，现在还是显示免费版

### DISCUSSION THREAD

**Discussion by Lok-D** [getlantern/lantern-forum-cn#786] [Q&A] [answered]
账号：50552243@qq.com
买了两次，现在都不能用，不行就先都给我退了吧
![QQ图片20240814222658](https://github.com/user-attachments/assets/688e405f-cbfd-4a77-a254-c56fc4524c56)
![QQ图片20240814222712](https://github.com/user-attachments/assets/334c6809-14b6-4962-a9f6-34d99677a762)

**Comment by MaxMKMax** [comment, marked as answer]
@Lok-D 您好. 已经帮您提交问题上去了, 请放心我们会帮您解决问题的. 谢谢您支持蓝灯.

**Reply by Lok-D** [reply]
能不能帮我退一个，只留一个就可以。

**Reply by Lok-D** [reply]
@Mike777Khoo 怎么样呢？现在什么情况？

**Reply by MaxMKMax** [reply]
@Lok-D 您好. 您的问题已经解决了. 帮您升级好专业版帐号了. 2 年. 您可以用专业版了.

**Reply by Lok-D** [reply]
@Mike777Khoo 现在是能用了，但是时间比我之前的要短，赠送的时间被扣了

**Reply by Lok-D** [reply]
@Mike777Khoo 赠送时间怎么算？补三个月把~

**Comment by alala1314** [comment]
解决了吗？

**Reply by MaxMKMax** [reply]
@alala1314 您好. 请问您是遇到什么问题吗? 请您提供我支付单据和账号邮箱.

**Reply by Lok-D** [reply]
@alala1314 能用了

**Reply by alala1314** [reply]
几天能用的我这还不行

**Reply by Lok-D** [reply]
@alala1314 三五天吧~管理员提交了之后

**Reply by wander0324** [reply]
> @alala1314 您好. 请问您是遇到什么问题吗? 请您提供我支付单据和账号邮箱.

你好，我8月16日购买的1年专业版，账号85423166@qq.com，订单号2024081622001316131454810921，目前的问题是登不上专业版，依然显示免费版，麻烦帮忙查看并解决呢，谢谢！
